client/extensions/woocommerce/state/sites/data/counts/selectors.js

```diff
@@ -1,61 +1,64 @@
-/**
- * External dependencies
- */
-import { get, reduce } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { statusWaitingPayment, statusWaitingFulfillment } from 'woocommerce/lib/order-status';
 
+function getItems( state, siteId ) {
+	return state?.extensions?.woocommerce?.sites?.[ siteId ]?.data?.counts?.items ?? {};
```

Comment:
Judging by how reducers are created, seems like if there's a site with that site ID, `.data.counts.items` will exist for it. Are these optional chaining cases unnecessary?

Reply:
Yes, I totally missed these. Thanks!